baybe/constraints/base.py

```diff
@@ -173,11 +175,10 @@ def to_botorch(
             if p in param_names
         ]
 
-        # TODO: Cast rhs to correct precision once BoTorch also supports single point.
         return (
             torch.tensor(param_indices),
             torch.tensor(self.coefficients, dtype=DTypeFloatTorch),
-            self.rhs,
+            np.asarray(self.rhs, dtype=DTypeFloatNumpy).item(),
```

Comment:
I think this is rather cumbersome. Why not simply `DTypeFloatNumpy(self.rhs)`?

Reply:
mypy (complaining you suggestion returns `object`)
happy to sue an alternative if you want (cast? ignore?) let me know

Reply:
oh, OK, that's unexpected and bad. Then I have no strong preference here, proceed as you wish :D